src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/generated.py

```diff
@@ -54,11 +54,11 @@
 cli_command('webapp deployment user show', ProviderOperations.get_publishing_user, factory)
 
 factory = lambda _: web_client_factory().server_farms
-cli_command('webapp plan create', create_app_service_plan)
-cli_command('webapp plan update', update_app_service_plan)
-cli_command('webapp plan delete', ServerFarmsOperations.delete_server_farm, factory)
-cli_command('webapp plan list', ServerFarmsOperations.get_server_farms, factory)
-cli_command('webapp plan show', ServerFarmsOperations.get_server_farm, factory)
+cli_command('app-service-plan create', create_app_service_plan)
```

Comment:
>app-service-plan  [](start = 13, length = 17)

what is our process for reviewing top-level items?  I feel like we are starting to lose our tidy set of top-level categories that differentiate us from the competition.  Web app is clear to anyone working with web, but app-service-plan is less obvious and now web app has 2 top-level entries.  We should discuss having our own "API reviews" especially for top-level items